releases/__init__.py

```diff
@@ -598,8 +598,8 @@ def unknown_visit(self, node):
 
 def generate_changelog(app, doctree):
     # Don't scan/mutate documents that don't match the configured document name
-    # (which by default is changelog.rst).
-    if app.env.docname != app.config.releases_document_name:
+    # (which by default is ['changelog.rst', ]).
+    if app.env.docname not in app.config.releases_document_name:
```

Comment:
I keep thinking on this. I like the idea of being able to simply specify a range of documents, but regex can be a pain to debug at the best of times.

Since this is passed from the Sphinx configuration file, which can execute arbitrary Python, would it make sense to have the user generate their own list somehow? Maybe add an example to the documentation.

Another option would be to pass it through a glob-er (is that even a word?) rather than a regex. Maybe this is done internally, maybe it is pushed to the config file. For example:

```
from pathlib import Path
p = Path('.')
list(p.glob(**/changelog.rst))
```